python/spear/config/default_config.vehicle.yaml

```diff
@@ -0,0 +1,46 @@
+#
+# Copyright(c) 2022 Intel. Licensed under the MIT License <http://opensource.org/licenses/MIT>.
+#
+
+VEHICLE:
+  VEHICLE_PAWN:
+    SKELETAL_MESH: "/Vehicle/OpenBot/Meshes/OpenBot_Mesh.OpenBot_Mesh"
```

Comment:
I still don't think the files in this folder are named correctly. (See my previous comment in the PR conversation)

Reply:
Renamed assets as we discussed - follow the convention in https://docs.unrealengine.com/5.2/en-US/recommended-asset-naming-conventions-in-unreal-engine-projects/